rllib/algorithms/impala/tf/vtrace_tf_v2.py

```diff
@@ -115,28 +115,6 @@ def vtrace_tf2(
     """
     log_rhos = target_action_log_probs - behaviour_action_log_probs
 
-    discounts = tf.convert_to_tensor(discounts, dtype=tf.float32)
```

Comment:
These seemed to be all superfluous. We are now only performing the for-loop on the CPU. Everything else (even within the vtrace_v2 functions) remains on the GPU.